### DISCUSSION TITLE
Numbering of books in series

### DISCUSSION THREAD

**Discussion by Piotr-GitHub** [advplyr/audiobookshelf#713] [Q&A]
Hi all,

I am not sure if it's issue with my deployment but I don't have any way to number the audiobooks in a series. I can only add manually number to a title. Is this feature removed or I am missing something?

Thanks!

**Comment by advplyr** [comment]
What version is the server?

**Comment by Piotr-GitHub** [comment]
Server runs on docker. Version 2.0.21. I've noticed that series that I have uploaded quite a long time ago has the numbers but a recently added audiobooks don't.

**Reply by advplyr** [reply]
You will need to share your exact folder structure and naming

**Reply by Piotr-GitHub** [reply]
Folder structure is automatically generated. But I think I know where is my error... I need to click edit on Series after the audiobook is uploaded to get the sequence numbering. It's a bit hidden and I missed it. Sequence number was earlier a separate field. I did not noticr it in changelog. Big sorry for the trouble.
